config.yaml

```diff
@@ -261,6 +261,27 @@ docusaurus:
                               - Overflow in C/: overflow-in-c/
                               - Overwrite Return Address/: overwrite-ret-addr/
                               - Overflow for Binary File/: overflow-for-binary/
+      - Lab 12 - Linking:
+          - File Linking:
```

Comment:
You can remove the `File Linking` subchapter since it's the only one in this lab. Go directly with `path` and the subsections.